### ISSUE TITLE
Requires an unreleased version of Factory Boy

### ISSUE THREAD

**Issue by ryneeverett** [rbarrois/django-factory_boy#3]
When attempting a pip install from git master:

```
Collecting factory-boy>=2.6.0 (from django-factory-boy==0.1.7)
  Could not find a version that satisfies the requirement factory-boy>=2.6.0 (from django-factory-boy==0.1.7) (from versions: 1.0.0, 1.0.4, 1.1.0, 1.1.1, 1.1.2, 1.1.3, 1.1.4, 1.1.5, 1.2.0, 1.3.0, 2.0.0, 2.0.1, 2.0.2, 2.1.0, 2.1.1, 2.1.2, 2.2.0, 2.2.1, 2.3.0, 2.3.1, 2.4.0, 2.4.1, 2.5.0, 2.5.1, 2.5.2)
```

**Comment by rbarrois** [comment]
Yes indeed, I plan to release the latest ``factory_boy`` in the upcoming days.

**Comment by rbarrois** [comment]
Fixed: I've just released version 1.0.0.

Looks like my "upcoming days" have turned into months :/

**Comment by johnraz** [comment]
:+1:
